README.md

```diff
@@ -453,22 +453,22 @@ The name of resources, and later the name of task outputs, determines the name u
 So, `hello-world` can access anything from `resource-tutorial` (this tutorial's `git` repository) under the `resource-tutorial/` path. Since the relative path of `task_hello_world.yml` task file inside this repo is `01_task_hello_world/task_hello_world.yml`, the `task: hello-world` references it by joining the two: `file: resource-tutorial/01_task_hello_world/task_hello_world.yml`
 
 
-There is a benefit and a downside to abstracting tasks into YAML files outside of the pipeline.
+There are benefits and a downsides to abstracting tasks into YAML files outside of the pipeline.
```

Comment:
Probaby you want "There are benefits and downsides" and not "There are benefits and a downsides"